README.md

```diff
@@ -39,6 +39,7 @@ git clone https://github.com/binast/binjs-ref
 cargo run --bin binjs_encode -- --help
 cargo run --bin binjs_decode -- --help
 ```
+Note: If you need to increase the memory for the node process by setting the environment variable `NODE_MAX_OLD_SPACE_SIZE=2048` (memory in MB, similar to node's `--max_old_space_size`). By default it uses 2048MB.
```

Comment:
I'd rephrase

```markdown
**Note** The JS parser may choke on very large JS source files. If so, you'll need to set the environment variable `NODE_MAX_OLD_SPACE_SIZE=xxxx`. This will instruct the Node-based parser to allocate more memory. The default value is 2048 (Mb). This is equivalent to passing `--max_old_space_size` to the Node process.
```